hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NetworkTopologyServlet.java

```diff
@@ -152,8 +153,8 @@ private void printJsonFormat(PrintStream stream, Map<String,
     }
   }
 
-  private void printTextFormat(PrintStream stream, Map<String,
-      TreeSet<String>> tree, ArrayList<String> racks) {
+  protected void printTextFormat(PrintStream stream, Map<String,
+          TreeSet<String>> tree, ArrayList<String> racks) {
```

Comment:
One very minor comment, there are unnecessary spaces.

Reply:
Thanks @tasanuma for your review, I fixed it.